examples/PersonSchema/personinfo/sqlschema/personinfo.sql

```diff
@@ -1,107 +1,107 @@
+--     * Slot: duration
+--     * Slot: is_current
 -- # Class: "Concept" Description: ""
---     * Slot: id Description:
---     * Slot: name Description:
---     * Slot: description Description:
---     * Slot: image Description:
+--     * Slot: id
+--     * Slot: name
+--     * Slot: description
+--     * Slot: image
 -- # Class: "DiagnosisConcept" Description: ""
---     * Slot: id Description:
---     * Slot: name Description:
---     * Slot: description Description:
---     * Slot: image Description:
+--     * Slot: id
+--     * Slot: name
+--     * Slot: description
+--     * Slot: image
 -- # Class: "ProcedureConcept" Description: ""
---     * Slot: id Description:
---     * Slot: name Description:
---     * Slot: description Description:
---     * Slot: image Description:
+--     * Slot: id
+--     * Slot: name
+--     * Slot: description
+--     * Slot: image
 -- # Class: "Relationship" Description: ""
---     * Slot: id Description:
---     * Slot: started_at_time Description:
---     * Slot: ended_at_time Description:
---     * Slot: related_to Description:
---     * Slot: type Description:
+--     * Slot: id
+--     * Slot: started_at_time
+--     * Slot: ended_at_time
+--     * Slot: related_to
+--     * Slot: type
 -- # Class: "FamilialRelationship" Description: ""
---     * Slot: id Description:
---     * Slot: started_at_time Description:
---     * Slot: ended_at_time Description:
---     * Slot: related_to Description:
---     * Slot: type Description:
+--     * Slot: id
+--     * Slot: started_at_time
+--     * Slot: ended_at_time
+--     * Slot: related_to
+--     * Slot: type
 --     * Slot: Person_id Description: Autocreated FK slot
 -- # Class: "EmploymentEvent" Description: ""
---     * Slot: id Description:
---     * Slot: employed_at Description:
---     * Slot: started_at_time Description:
---     * Slot: ended_at_time Description:
---     * Slot: duration Description:
---     * Slot: is_current Description:
+--     * Slot: id
+--     * Slot: employed_at
+--     * Slot: started_at_time
+--     * Slot: ended_at_time
+--     * Slot: duration
+--     * Slot: is_current
 --     * Slot: Person_id Description: Autocreated FK slot
 -- # Class: "MedicalEvent" Description: ""
---     * Slot: id Description:
---     * Slot: in_location Description:
---     * Slot: started_at_time Description:
---     * Slot: ended_at_time Description:
---     * Slot: duration Description:
---     * Slot: is_current Description:
+--     * Slot: id
+--     * Slot: in_location
+--     * Slot: started_at_time
+--     * Slot: ended_at_time
+--     * Slot: duration
+--     * Slot: is_current
 --     * Slot: Person_id Description: Autocreated FK slot
---     * Slot: diagnosis_id Description:
---     * Slot: procedure_id Description:
+--     * Slot: diagnosis_id
+--     * Slot: procedure_id
 -- # Class: "WithLocation" Description: ""
---     * Slot: id Description:
---     * Slot: in_location Description:
+--     * Slot: id
+--     * Slot: in_location
```

Comment:
It would be nice to make the representation of the name and description in the class and the slot uniform -- i.e. either both should be quoted or both should not be. Similarly, if there is no description given for the class, omit the "Description" field (or add it back for the slot and have the contents as `""`). It looks rather haphazard as-is.